optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/main/UpgradeProcedureTest.java

```diff
@@ -114,9 +114,8 @@ public void validationIsDoneBeforeUpgradeExecution() {
 
     // then the validation and execution happens in the expected order
     final InOrder inOrder = inOrder(validationService, createIndexStep);
-    // The validation order matters since we first need to ensure that the ES client
+    // The validation order matters since we first need to ensure that the DB client
     // is able to communicate to ElasticSearch before using it to retrieve the schema version.
```

Comment:
In theory this could be changed to 
```suggestion
    // is able to communicate to the database before using it to retrieve the schema version.
```
but since we are mocking an ES database anyway, we can leave it as is